src/olympia/signing/tests/test_views.py

```diff
@@ -514,11 +514,6 @@ def setUp(self):
         AppVersion.objects.create(application=amo.FIREFOX.id, version='42.0')
         AppVersion.objects.create(application=amo.FIREFOX.id, version='*')
 
-        validate_patcher = mock.patch('validator.validate.validate')
-        run_validator = validate_patcher.start()
-        run_validator.return_value = json.dumps(amo.VALIDATOR_SKELETON_RESULTS)
-        self.addCleanup(validate_patcher.stop)
```

Comment:
This was used to avoid `dump_apps` to influence the tests here but that shouldn't be a problem anymore since the recent conftest / tmpdir refactor and I haven't seen a related test failure while working on this for a while.